### ISSUE TITLE
Test Remote-SSH: Use RemoteCommand in connection.

### ISSUE THREAD

**Issue by tanhakabir** [microsoft/vscode-remote-release#6221]
Refs: https://github.com/microsoft/vscode-remote-release/issues/4474

- [x] macOS @lszomoru 
- [x] linux @JacksonKearl 
- [x] windows @rzhao271 

Complexity: 3

[Create Issue](https://github.com/microsoft/vscode-remote-release/issues/new?body=Testing+%236221%0A%0A&assignees=tanhakabir)

---

We want to make sure 

Prerequisites: 
1. Use the [pre-release of the Remote-SSH extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh)
2. SSH into your own machine: [Windows](https://github.com/microsoft/vscode-remote-release/wiki/Sanity-Check-VS-Code-Servers#windows), [macOS](https://github.com/microsoft/vscode-remote-release/wiki/Sanity-Check-VS-Code-Servers#macos), and [Linux](https://github.com/roblourens/ssh-containers) to get one set up locally.

Steps:

1. Set `remote.SSH.useLocalServer` to `true` and `remote.SSH.enableRemoteCommand` to `true`.
2. Edit your SSH config for your remote to include `RemoteCommand` field with switching user with `su`.
2. Connect to your remote.
3. Verify you can successfully connect.
4. Check that terminal starts with the switched user.

**Comment by lszomoru** [comment]
@tanhakabir, could you please provide a sample SSH config? Thank you!

**Comment by tanhakabir** [comment]
This is a config I used to connect:
```
Host second
  HostName 147.182.250.143
  User root
  RemoteCommand su test
```

**Comment by rzhao271** [comment]
I had some trouble testing the feature on Windows, but after discussing with Tanha, we got it. The following might help for other OSes too:

- I explicitly set `"remote.SSH.useLocalServer": true` in my settings file.
- For RemoteCommand, I used `RemoteCommand bash -c "export TESTKEY=testvalue; bash"`, and then in the bash terminal on the opened SSH window, I ran `env | grep TEST` and confirmed that that environment variable was set.
